source/ship/ship.gd

```diff
@@ -36,8 +36,8 @@ func visit_starport() -> void:
 
 func _on_area_2d_input_event(
 	_viewport: Node,
-	event: InputEventMouseButton,
+	event: InputEvent,
```

Comment:
Can you explain the reasoning behind this change? I put it to `InputEventMouseButton` because then it only seemed to run those events that i was interested in. Now I think it will run on MouseMotion events as well? So doesn't this run multiple times you hold mouse button left and move mouse over the area?